### ISSUE TITLE
[Feature] Add Checkboxes Category and several Components

### ISSUE THREAD

**Issue by ShravaniPathak** [Rakesh9100/Beautiify#1070]
### Description

In UI design, a checkbox appears as a little square box on the screen that the user can check or uncheck. A checkbox allows users to select one of multiple options from a list, with each checkbox operating as an individual. 

A checkbox can greatly improve user-friendliness and encourage user-interaction

### Screenshots

![checkbox](https://github.com/Rakesh9100/Beautiify/assets/140315107/f2b6e00d-e6eb-49e8-8e1c-857e134be85d)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by Rakesh9100** [comment]
What exactly do you want to add and where @ShravaniPathak

**Comment by ShravaniPathak** [comment]
I noticed that there are a list of components present on the home page (such as buttons, cards, etc), that are useful when building a webpage. I wanted to "**checkbox**" to this list, and I was hoping to add an **interactive square checkbox** that looks like the embedded screenshot within this component:
![checkbox](https://github.com/Rakesh9100/Beautiify/assets/140315107/073711a6-a491-4e75-84fe-b04521cf0bad)

**Comment by Rakesh9100** [comment]
Okay, so you want to add Checkbox Category. Will you also add 2-3 components inside that @ShravaniPathak

**Comment by ShravaniPathak** [comment]
Yes. I'm thinking of adding :
1. An interactive square checkbox that allows user to select multiple options
![checkbox](https://github.com/Rakesh9100/Beautiify/assets/140315107/38cb9559-ffeb-41bc-adcf-6b53ef1f5036)
2. A interactive dot checkbox that only allows user to select a single option
![radio-buttons](https://github.com/Rakesh9100/Beautiify/assets/140315107/187cd97d-ac4c-4425-a43d-aa5da7103966)

**Comment by Rakesh9100** [comment]
The second one seems radio button and not a checkbox, correct me if I am wrong @ShravaniPathak

**Comment by ShravaniPathak** [comment]
Yes, my apologies. I accidently attached the wrong image. Here is the circular checkbox that I would like to implement (that would only allow one option to be selected at a time):
![circular_check](https://github.com/Rakesh9100/Beautiify/assets/140315107/845e98ed-27d7-4489-a577-a65688d3e433)

**Comment by Rakesh9100** [comment]
Okay go ahead @ShravaniPathak

**Comment by Rakesh9100** [comment]
Pls provide your PR @ShravaniPathak

**Comment by Sakshamjain98** [comment]
Assign me this under iwoc

**Comment by Rakesh9100** [comment]
Just don't say to assign in every issue, comment properly with the idea of how will you solve a particular issue @Sakshamjain98
